pkgs/by-name/uh/uhhyou-plugins-juce/package.nix

```
@@ -0,0 +1,73 @@
+{
+  alsa-lib,
+  cmake,
+  curl,
+  fetchFromGitHub,
+  freetype,
+  lib,
+  libGL,
+  libXcursor,
+  libXext,
+  libXinerama,
+  libXrandr,
+  libjack2,
+  pkg-config,
+  stdenv,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "uhhyou-plugins-juce";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "ryukau";
+    repo = "UhhyouPluginsJuce";
+    tag = "UhhyouPluginsJuce${finalAttrs.version}";
+    hash = "sha256-oHxyOTiqEwdNUKGQNjjfdRkzMa+4TYX6Vf6ZS9BTcC0=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+  buildInputs = [
+    alsa-lib
+    curl
+    freetype
+    libGL
+    libXcursor
+    libXext
+    libXinerama
+    libXrandr
+    libjack2
+  ];
+
+  # Disable LTO to avoid optimization mismatch issues
+  NIX_CFLAGS_COMPILE = [
+    "-fno-lto"
+  ];
+
+  passthru.plugins = [
+    "TwoBandStereo"
+    "SlopeFilter"
+    "EasyOverdrive"
+    "AmplitudeModulator"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/vst3
+    ${lib.concatMapStringsSep "\n" (plugin: ''
+      cp -r ${plugin}_artefacts/Release/VST3/${plugin}.vst3 $out/lib/vst3/
+    '') finalAttrs.finalPackage.passthru.plugins}
```

Hmm you could still use a shell `*` glob pattern:


```suggestion
    cp -r *_artefacts/Release/VST3/*.vst3 $out/lib/vst3/
```

It should work.